doc/ranger.pod

```diff
@@ -192,7 +192,8 @@ scripts by setting the option C<use_preview_script> and C<preview_files> to true
 This default script is F<%rangerdir/data/scope.sh>. It contains more
 documentation and calls to the programs I<lynx> and I<elinks> for html,
 I<highlight> for text/code, I<img2txt> for images, I<atool> for archives,
-I<pdftotext> or I<mutool> for PDFs and I<mediainfo> for video and audio files.
+I<pdftotext> or I<mutool> for PDFs, or I<mediainfo> for video and audio files
+and I<openscad> for 3D models.
```

Comment:
Hmm, the listing here isn't complete anyway. I'm not sure we should add openscad since it's relatively obscure.

Reply:
Well, need a decision here. If its up to me, I'd list it. But you decide :smile:

Reply:
I'd go with no unless @Vifon feels differently.

Reply:
It might be good for "marketing". Showing neat things ranger can do won't hurt. Let's go for it.

Reply:
@Vifon, does this mean you'd like us to put *everything* up there? Marketing's for the README imo, this is the manpage, people would've already installed ranger by this point, no? It just seems little silly to make it harder to find how to get openscad previews as more mentions means manpage search shows more irrelevant results.

Reply:
Ah, it's a manpage? Still, previewing 3D models in particular is pretty neat. Definitely not everything but I'd include this one.

Reply:
OK, let's move it to the README.

Reply:
@Vifon it already was/is in the README : >